app/routes/endemics/index.js

```diff
@@ -53,7 +54,7 @@ module.exports = {
         session.setEndemicsClaim(request, referenceKey, tempClaimId)
 
         // new user
```

Comment:
This comment isn't really true any more

Reply:
We've changed approach, all changes in this PR have been reverted. Thanks for reading over it.